fixtures/eurolite/led-bar-12-qcl-rgbw.json

```diff
@@ -0,0 +1,235 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Led bar-12 QCL RGBW",
+  "categories": ["Color Changer", "Dimmer", "Pixel Bar"],
```

Comment:
This fixture is not a pixel bar as it doesn't have individually controllable leds